modules/indexing/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/filtering/PropPath.scala

```diff
@@ -82,6 +90,10 @@ object PropPath extends PropPathBuilder {
     case PathZeroOrOne(uri)                => s"(${uri.show})?"
     case SeqPath(first, second)            => s"${showPath.show(first)}/${showPath.show(second)}"
     case AlternativeSeqPath(first, second) => s"${showPath.show(first)}|${showPath.show(second)}"
+    case NegatedSeqPath(Nil)               => ""
+    case NegatedSeqPath(list) =>
+      list.foldLeft(Vector.empty[String])((acc, current) => acc :+ showPath.show(current)).mkString("!(", "|", ")")
```

Comment:
format folds with curlies, easier to read:

```
  val paths = list.foldLeft(Vector.empty[String]) {
    case (acc, el) => acc :+ current.show
  }
  paths.mkstring("!(", "|", ")")
```

Reply:
Done